code/game/objects/items/stacks/stack.dm

```diff
@@ -246,6 +246,11 @@
 				continue
 			if(istype(AM,/obj/structure/table))
 				continue
+			if(istype(AM,/obj/structure/window))
+				var/obj/structure/window/W = AM
+				var/obj/structure/window/O = new R.result_type
```

Comment:
What are you trying to do here, you're creating random objects here in nullspace on every build check.